include/matx/core/tensor_utils.h

```diff
@@ -56,23 +62,44 @@ namespace matx
     return 0;
   }
 
+
+  /**
+   * @brief finds the size of the largest dim of the tensor
+   *core/tensor_utils.h
+   * @param op Operator
+   * @return size of largest dim
+   */
+  template <typename Op>
+  index_t LargestDimSize(const Op &op) {
+    index_t maxSize = 0;
+    for (int i = 0; i < op.Rank(); i++)
+    {
+      if( op.Size(i) > maxSize)
```

Comment:
See if 
```
maxSize = cuda::std::max(op.Size(i), maxSize);
```
I don't remember if it's in cuda::std yet

Reply:
cuda:std::max isnt' there, but std::max is and is supported for these types, so changed to use that